pkg/metadata/host/payload.go

```diff
@@ -26,6 +26,7 @@ type Meta struct {
 	Hostname       string   `json:"hostname"`
 	HostAliases    []string `json:"host_aliases"`
 	InstanceID     string   `json:"instance-id"`
+	AgentHostname  string   `json:"agent-hostname"`
```

Comment:
should we add `omitempty` here? Or is it OK for the backend if the key is always there, but most of the time with an empty value? (i.e. does the backend properly ignore the key when it's present with an empty value?)

Reply:
The backend handles empty value correctly but adding `omitempty` emphases on the fact this field is optional. Added it.